analysis/topeft_run2/analysis_processor.py

```diff
@@ -1074,15 +1115,15 @@ def process(self, events):
                                             if (("ptz" in dense_axis_name) & ("onZ" not in lep_chan) & ("offZ_high" not in lep_chan) & ("offZ_low" not in lep_chan)):continue
                                         elif self.tau_h_analysis:
                                             if (("ptz" in dense_axis_name) and ("onZ" not in lep_chan)): continue
-                                            if (("ptz" in dense_axis_name) and ("2lss" not in lep_chan) and ("ptz_wtau" not in dense_axis_name)): continue
-                                            if (("ptz_wtau" in dense_axis_name) and ("1tau" not in lep_chan)): continue
+                                            if (("ptz" in dense_axis_name) and ("2lss" in lep_chan) and ("ptz_wtau" not in dense_axis_name)): continue
```

Comment:
Was the `"2lss" not in lep_chan` incorrect before?

Reply:
It was, this is to avoid saving the traditional `ptz` in `2lss` regions, but saving `ptz_wtau` for these regions